playbooks/robusta_playbooks/kubectl_enrichments.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import uuid
+
+from hikaru.model.rel_1_26 import Container, PodSpec
+from robusta.api import (
+    RUNNER_SERVICE_ACCOUNT,
+    ExecutionBaseEvent,
+    FileBlock,
+    MarkdownBlock,
+    PodRunningParams,
+    RobustaJob,
+    action,
+)
+from robusta.utils.parsing import format_event_templated_string
+
+IMAGE: str = os.getenv("KUBECTL_IMAGE_OVERRIDE", f"bitnami/kubectl:latest")
+
+
+class KubectlParams(PodRunningParams):
+    """
+    :var kubectl_command: The full kubectl command to run, formatted as a shell command string.
+    :var description: A description of the command ran.
+    :var timeout: The maximum time (in seconds) to wait for the kubectl command to complete. Default is 3600 seconds.
+    """
+
+    command: str = None  # type: ignore
```

Comment:
Why type ignore? Just need to wrap str with Optional no?

Reply:
its not optional, whats a kubectl_command without the actual kubectl command to run?